flang/lib/Semantics/resolve-directives.cpp

```diff
@@ -2152,6 +2174,114 @@ void OmpAttributeVisitor::CollectNumAffectedLoopsFromClauses(
   }
 }
 
+void OmpAttributeVisitor::CheckPerfectNestAndRectangularLoop(
+    const parser::OpenMPLoopConstruct &x) {
+  auto &dirContext = GetContext();
```

Comment:
```suggestion
  auto &dirContext{GetContext()};
```
nit: the style in semantics is to use braced initialisation